### PR TITLE
Port to Webkit2

### PR DESCRIPTION
**Pull request by cemeiq**
Port to Webkit2

### CODE REVIEW

**Comment by iqraceme** [comment]
@quozl  please, review this.

**Comment by chimosky** [comment]
Reviewed, thanks. Remove the two new lines from the `activity.info` file, as they're not needed.

**Comment by iqraceme** [comment]
I have removed the two new lines from activity.info file @chimosky . Thanks

**Comment by quozl** [comment]
Reviewed to cfe490c.  I'm curious.  What's the vcs.xml file for?

**Comment by iqraceme** [comment]
Its Pycharm IDE's configuration file. Apologies. Removing it

**Comment by quozl** [comment]
Sorry for the delay, slowly getting through my 3500 mail messages.  Was about to merge, but;

> I have removed the two new lines from activity.info file @chimosky . Thanks

Please `git add` and `git commit` that?  At the moment the pull request [files changed](https://github.com/sugarlabs/grid-paint-activity/pull/4/files) tab shows the added lines.

File: enyo.py [hunk 1]
```diff
@@ -25,7 +25,6 @@ def connect(self, name, callback):
 
     def send_message(self, name, args=None):
         """Send a message to Enyo"""
-        script = "enyo.Sugar.sendMessage('" + name + "', "
```
Comment (quozl) [inline enyo.py]: A few lines later the variable script is referenced again; won't this generate an error?

Comment (iqraceme) [reply enyo.py]: Let me fix this. Thanks !

**Pull request merged**